.danger/major-missing-upgrade-guide.ts

```diff
@@ -0,0 +1,51 @@
+import * as fs from 'fs';
+import {getChangesetsFromFiles} from './utils';
+// you kind of have to treat Danger plugins as global?
+// https://danger.systems/js/usage/extending-danger.html#writing-your-plugin
+import {DangerDSLType} from 'danger/distribution/dsl/DangerDSL';
+declare const danger: DangerDSLType;
+export declare function fail(message: string): void;
+
+/** location in the repo where the upgrade guide lives */
+export const UPGRADE_GUIDE_PAGE_FILE = 'packages/paste-website/src/pages/core/upgrade-guide.mdx';
+
+export const FAIL_MESSAGE = `You have marked a Paste package as a MAJOR update via changeset, but have not provided a corresponding [upgrade guide](https://paste.twilio.design/core/upgrade-guide) entry.
+Please update the guide to inform our users what to do in response to this major update to Paste.`;
+
+/**
+ * Check if the changeset file contents include a mention of a package major update
+ *
+ * @param {string} filePath
+ * @return {*}
+ */
+export const hasMajorUpdate = (filePath: string) => {
```

Comment:
suggestion: we are now using both "upgrade" and "update" here. Is there a reason you changed it from "hasMajorUpgrade" to "hasMajorUpdate"? curious to know what the distinction is, and if there's no difference, I'd recommend we use consistent language for clarity

Reply:
ok, thanks for the clarification. makes sense!